src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Automatic-Module-Name: io.vertx.core
+
```

Comment:
Note: The new line is required. Otherwise the maven plugin does not work for some reason.

Reply:
shall we upgrade the plugin version ?

Reply:
I mean would it solve the issue ?

Reply:
I tried with the newest I could find on maven central (maven jar plugin 3.2.0) and the issue is still present

Reply:
There is no error or anything, the jar is built but the Automatic-Module-Name property is not in the built jar.

Reply:
ok then